specification/common-types/resource-management/v3/managedidentity.json

```diff
@@ -11,7 +11,8 @@
       "description": "The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.",
       "type": "object",
       "additionalProperties": {
-        "$ref": "#/definitions/UserAssignedIdentity"
+        "$ref": "#/definitions/UserAssignedIdentity",
+        "x-nullable": true
       }
```

Comment:
revert. pl dont change the existing common types as this will be a breaking change for a lot of services.